src/System.Net.NetworkInformation/tests/FunctionalTests/IPInterfacePropertiesTest_Linux.cs

```diff
@@ -0,0 +1,179 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Net.Sockets;
+using System.Net.Test.Common;
+
+using Xunit;
+using Xunit.Abstractions;
+
+namespace System.Net.NetworkInformation.Tests
+{
+    [PlatformSpecific(PlatformID.Linux)]
+    public class IPInterfacePropertiesTest_Linux
+    {
+        private readonly ITestOutputHelper _log;
+
+        public IPInterfacePropertiesTest_Linux()
+        {
+            _log = TestLogging.GetInstance();
+        }
+
+        [Fact]
+        public void IPInfoTest_AccessAllProperties_NoErrors()
+        {
+            foreach (NetworkInterface nic in NetworkInterface.GetAllNetworkInterfaces())
+            {
+                _log.WriteLine("Nic: " + nic.Name);
+                _log.WriteLine("- Supports IPv4: " + nic.Supports(NetworkInterfaceComponent.IPv4));
+                _log.WriteLine("- Supports IPv6: " + nic.Supports(NetworkInterfaceComponent.IPv6));
+
+                IPInterfaceProperties ipProperties = nic.GetIPProperties();
+
+                Assert.NotNull(ipProperties);
+
+                Assert.Throws<PlatformNotSupportedException>(() => ipProperties.AnycastAddresses);
+
+                Assert.NotNull(ipProperties.DhcpServerAddresses);
+                _log.WriteLine("- Dhcp Server Addresses: " + ipProperties.DhcpServerAddresses.Count);
+                foreach (IPAddress dhcp in ipProperties.DhcpServerAddresses)
+                {
+                    _log.WriteLine("-- " + dhcp.ToString());
```

Comment:
This just uses the System.Net logging code in Common\src\System\Net\Logging. From what I understand, it doesn't actually write to any output files, but writes to an internal event source. I don't know if those events can be read x-plat (I'm not attempting to read them or anything), so I wouldn't have a problem with removing the logging. On the other hand, it doesn't really hurt to just maintain the same format as the Windows tests, and the logging can probably be enabled, or implemented in another way (just make a simple ITestOutputHelper for Unix).